src/catalua.cpp

```diff
@@ -1024,6 +1035,17 @@ static int game_dofile(lua_State *L)
     return 0;
 }
 
+static int game_myPrint(lua_State *L)
+{
+    int argc = lua_gettop(L);
+    for( int i = argc; i > 0; i--) {
+        std::string param = lua_tostring_wrapper(L, -i);
```

Comment:
This temporary variable seems a bit useless. Why isn't the result of the function directly forwarded to the stream?